pkg/domain/entities/types.go

```diff
@@ -122,6 +123,12 @@ type BuildOptions struct {
 type BuildReport struct {
 	// ID of the image.
 	ID string
+	// Format to save the image in
+	SaveFormat string
+}
+
+type FarmBuildOptions struct {
+	Cleanup bool
```

Comment:
This field needs a description of some kind.

Reply:
fixed